pom.xml

```diff
@@ -15,6 +15,11 @@
     
     <repositories>
         <repository>
+          <id>central</id>
+          <name>central</name>
+          <url>https://repo1.maven.org/maven2</url>
+        </repository>
+        <repository>
```

Comment:
Usually the central repo is in the global settings for the box. (from what i recall, i read about that somewhere)